charmcraft.yaml

```diff
@@ -130,3 +130,4 @@ parts:
     charm-binary-python-packages:
       - jsonschema
       - "pydantic>=2"
+      - "setuptools>=70.0.0"
```

Comment:
question: do we need to specify this here or can we move it to `requirements.txt`?

Reply:
didnt work when i put it in there